Week-7/Problem_7.5/nextGreat.js

```diff
@@ -0,0 +1,21 @@
+function nextGreaterElement(array, n) {
+  var next, i, j;
+  for (i = 0; i < n; i++) {
+    next = -1;
+    for (j = i + 1; j < n; j++) {
+      if (array[i] < array[j]) {
+        next = array[j];
+        break;
+      }
+    }
+    console.log(array[i] + " -- " + next);
+  }
+}
+
+var newArray = [11, 13, 21, 3];
+var n = newArray.length;
+nextGreaterElement(newArray, n);
```

Comment:
Please refer the implementation here to reduce the time complexicity to O(n)

https://www.geeksforgeeks.org/next-greater-element/#:~:text=the%20above%20approach%3A-,Follow%20the%20steps%20mentioned%20below%20to%20implement%20the%20idea%3A,-Push%20the%20first